### DISCUSSION TITLE
selecting a frequency step

### DISCUSSION THREAD

**Discussion by jh4awd** [egzumer/uvk5-chirp-driver#5] [Q&A]
Thank you for your great work.
I am trying to do memory channel editing using CHIRP_egzumer.exe.
When selecting a frequency step from the pull-down list, the first 2.5K to the eighth 0.01K in the list are entered correctly, but frequencies after that are not entered correctly.
For example, if you select 20KHz, 2.5KHz will be input.
I have a problem when setting up VFO.
Is there something wrong with my operating method or my understanding?

**Comment by egzumer** [comment]
fixed

**Reply by jh4awd** [reply]
confirmed.
Thank you for your quick response.
